### DISCUSSION TITLE
Does this mod download other mods?

### DISCUSSION THREAD

**Discussion by Handrail9** [Sinytra/ConnectorExtras#47] [Q&A] [answered]
Hello, I am trying to figure out what mod keeps redownloading continuity because I keep removing it and it keeps being added back. I noticed in the mods list in game that Extras has a continuity extra. If it isn't this mod that's redownloading it are there any ideas as to a mod that does download continuity? It's getting really difficult to test my modpack as I am choosing to use Embeddium + Embeddium++ which cause the game to crash when continuity is also installed.

**Comment by Su5eD** [comment, marked as answer]
No, connector does not download anything at runtime.
